src/io/flutter/inspector/InspectorSourceLocation.java

```diff
@@ -30,8 +31,13 @@ public VirtualFile getFile() {
       return parent != null ? parent.getFile() : null;
     }
 
-    // We have to strip the file:// prefix from the Dart file path for compatibility.
-    final String filePrefix = "file://";
+    // We have to strip the file:// or file:/// prefix depending on the
+    // operating to convert from paths stored as URIs to local file system
+    // paths.
+    // TODO(jacobr): remove this workaround after the code in package:flutter
+    // is fixed to return local file system paths instead of URIs.
```

Comment:
Do we want to change to file system paths (on both sides of this call) or `file:/` urls?